S/satsuma/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "satsuma"
+version = v"0.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/markusa4/satsuma", "be6beeb6d2538aa133b1f6b7cad84655cda950bb"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/satsuma
+
+for f in $WORKSPACE/srcdir/patches/*.patch; do
+    atomic_patch -p1 ${f}
+done
+
+cp $WORKSPACE/srcdir/tsl/* tsl/
+
+cmake .
+make satsuma
+install satsuma $bindir
```

Comment:
https://docs.binarybuilder.org/stable/build_tips/#CMake-builds

Reply:
I'm following the instructions from the package.

Reply:
You must pass the toolchain file (and also all the other options specified there if you want to do the right thing), whatever the instructions of the package may say because they ignore the fact people may need toolchain files.

Reply:
```suggestion
cmake -B build -DCMAKE_INSTALL_PREFIX=${prefix} -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release
cmake --build build --parallel ${nproc}
cmake --install build
```

Reply:
Well, the build worked before your suggested changes, but with the suggestion from #CMake-builds it doesn't anymore :(
First, the cmake --build builds the wrong target (dejavu) and not satsuma; second, the cmake --install doesn't install anything.
I kept the first incantation, but replaced the next two by a direct make and install call.